CMakeLists.txt

```diff
@@ -942,6 +942,15 @@ if (HDF5_ENABLE_MAP_API)
   set (H5_HAVE_MAP_API 1)
 endif ()
 
+#-----------------------------------------------------------------------------
+# Option to Build with Static CRT libraries on Windows
+#-----------------------------------------------------------------------------
+option (HDF5_BUILD_STATIC_CRT_LIBS "Build With Static Windows CRT Libraries" OFF)
```

Comment:
Seems like this option is something you only want to bring in when building under Windows. Otherwise it's just adding more options to the space that aren't going to get used.